### PR TITLE
[routing-utils] Allow to appendRoutes add null phase

### PR DESCRIPTION
**Pull request by tobiaslins**
### Related Issues

For Analytics V2 we need to append routes to the `null` phase instead of `filesystem`.

#### Tests

- [x] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] This PR has a concise title and thorough description useful to a reviewer
- [x] Issue from task tracker has a link to this PR

### CODE REVIEW

File: packages/routing-utils/src/types.ts [hunk 1]
```diff
@@ -123,5 +123,5 @@ export interface AppendRoutesToPhaseProps {
   /**
    * The phase to append the routes such as `filesystem`.
```
Comment (dglsparsons) [inline packages/routing-utils/src/types.ts:124]: Might be worth expanding this comment with an explanation of the behaviour of passing `null`

File: packages/routing-utils/test/append.spec.ts [hunk 2]
```diff
@@ -128,3 +128,41 @@ test('appendRoutesToPhase one routes before, two routes in phase, two routes in
...
+  const phase = null;
+  const actual = appendRoutesToPhase({ routes, newRoutes, phase });
+  const expected = [
+    { src: '/first', dest: '/one' },
+    { src: '/second', dest: '/two' },
+    { src: '/new', dest: '/to' },
+  ];
+
+  deepEqual(actual, expected);
+});
```
Comment (styfle) [inline packages/routing-utils/test/append.spec.ts:168]: Lets add one more test when `const routes = []`

File: packages/routing-utils/test/append.spec.ts [hunk 3]
```diff
@@ -128,3 +128,41 @@ test('appendRoutesToPhase one routes before, two routes in phase, two routes in
...
+
+test('appendRoutesToPhase to null phase with no handle', () => {
+  const routes: Route[] = [
+    { src: '/first', dest: '/one' },
+    { src: '/second', dest: '/two' },
+  ];
+  const newRoutes = [{ src: '/new', dest: '/to' }];
```
Comment (styfle) [inline packages/routing-utils/test/append.spec.ts:158]: Also lets and another test when `newRoutes` has two routes

File: packages/routing-utils/src/types.ts (hunk 1 above)
Comment (styfle) [inline packages/routing-utils/src/types.ts:124]: Lets updating the JSDoc comment here to mention that `null` represents the phase prior to any `handle` being found.

Comment (dglsparsons) [reply packages/routing-utils/src/types.ts:124]: :spiderman-pointing:

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**